pkg/disttask/importinto/encode_and_sort_operator.go

```diff
@@ -250,3 +257,10 @@ func getNumOfIndexGenKV(tblInfo *model.TableInfo) int {
 	}
 	return count
 }
+
+func getKVGroupBlockSize(group string) int {
```

Comment:
I still worried about the block size is not enough. In lightning's CSV parser, we can tolerate ~120M KV size (however the only case user get such a big KV pair is that line terminator is not correct / source file is corrupted so the multiple lines are parsed as one line)

https://github.com/pingcap/tidb/blob/b0c01fa9a44e0abbbed9abca306aefd29e16bda2/br/pkg/lightning/mydump/csv_parser.go#L44

And if we don't split `memLimit` into multiple blocks, we will not meet this problem. I don't have strong oponion about this suggestion

Reply:
discussed with @Benjamin2037 last week, plan to add a restriction in doc about kv size when import using global sort

> a big KV pair is that line terminator is not correct / source file is corrupted

for this case, if multiple line is concated and parsed successfully, we can identified it easily